yaki_admin_backend/src/main/java/com/xpeho/yaki_admin_backend/data/services/UserServiceImpl.java

```diff
@@ -55,4 +57,15 @@ public UserEntity findById(int id) {
                     userModel.getLogin());
         } else throw new EntityNotFoundException("Entity User with id " + id + " not found");
     }
+
+    @Override
+    public UserEntity deleteById(int id) {
+        final Optional<UserModel> userModelOpt = userJpaRepository.findById(id);
+        if (userModelOpt.isPresent()) {
+            UserModel userModel = userModelOpt.get();
+            userJpaRepository.deleteById(id);
+            return new UserEntity(userModel.getLastName(), userModel.getFirstName(),userModel.getEmail(),userModel.getLogin());
+        } else throw new EntityNotFoundException("The team with id " + id + " not found.");
+
+    }
```

Comment:
what is the link with authentication ?

this should be in another pull request